docs/platform/billing-and-support/billing-and-payments-new-manager/index.md

```diff
@@ -0,0 +1,206 @@
+
+The prorated transfer quota is reset at the beginning of the following month.
+
+## Accessing Billing History
+
+All of your billing history is stored in the Linode Manager. Here's how to access it:
+
+1.  Log in to the [Linode Manager](http://manager.linode.com).
+2.  Select **Account**.
+3.  Select **Account & Billing**.
+4.  Expand the **Recent Invoices** and **Recent Payments** boxes.
+
+Select an invoice to view the charges for a particular month. <!-- You can also download invoices in PDF format. -->
+
+## Referral Credits
+
+You can receive service credit by referring new users to Linode. When you refer someone who maintains at least one active Linode for 90 days, your account will be issued a $20 service credit. Here's how to find your account referral code and URL:
+
+1.  Log in to the [Linode Manager](http://manager.linode.com).
+2.  Select the **My Profile** link by clicking on your username at the top of the page.
+3.  Select the **Referrals** tab.
+4.  The referral code and URL are listed under the **Referrals** section. You can provide the code to friends and use the URL on your website to generate referrals.
+
+Referral service credits must be used to purchase Linode services, and cannot be refunded as cash.
+
+## Updating Credit Card Information
+
+Keep your credit card information up to date to prevent service interruptions. Here's how:
+
+1.  Log in to the [Linode Manager](http://manager.linode.com).
+2.  Select **Account**.
+3.  Select **Account & Billing**.
+4.  Expand the **Update Credit Card** box and enter your credit card number and the card's expiration date.
+5.  Click **Save**.
+
+Your credit card information will be updated.
+
+ {{< note >}}
+If you have an outstanding balance, you will need to make a manual payment to bring your account up to date. See the [Making a Payment](#making-a-payment) section for more information.
+{{< /note >}}
+
+## Removing Services
+
+Our services are provided without a contract, so you're free to remove services from your account at any time. Here's how:
+
+1.  Log in to the [Linode Manager](http://manager.linode.com).
+2.  To remove a Linode from your account, select **Linodes**. From the Lindode's menu select **Settings**. Expand the **Delete Linode** box and click **Delete**.
+3.  To remove a NodeBalancer from your account, select **NodeBalancers**. Open the menu of the NodeBalancer you would like to remove, then select **Remove**.
+4.  To remove the Linode Backup Service, select **Linodes**. Select the corresponding Linode. Under the **Backups** tab click the **Cancel Backups** button at the bottom of the page.
+
+<!--
+## Canceling Your Account
+
+You can cancel your account and receive a refund. Here's how:
+
+1.  Log in to the [Linode Manager](http://manager.linode.com).
```

Comment:
I think we do have to route people to the old manager for this, but it might be a good idea to let them know that's what we're doing.